### PR TITLE
install ansible via pip. pin version to 2.0.2.0

### PR DESCRIPTION
**Pull request by domdom82**

### CODE REVIEW

**Comment by markusthoemmes** [comment]
rebased and merged dac1ca7c35e48f702a6fab629a7ff1728d9b1445

File: tools/ubuntu-setup/pip.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 sudo apt-get install -y python-pip
 sudo pip install jsonschema
 sudo pip install argcomplete
-sudo pip install docker-py
```
Comment (csantanapr) [inline tools/ubuntu-setup/pip.sh]: Why move docker-py from pip.sh to ansible.sh?

Comment (markusthoemmes) [inline tools/ubuntu-setup/pip.sh]: It's essentially a dependecy of `ansible` in our build, hence it makes sense to install it in conjunction with ansible.

Comment (csantanapr) [inline tools/ubuntu-setup/pip.sh]: That's what I suspected, so why docker-py not installed before ansible? or at least make a comment if it doesn't matter  the order of the installation.

Comment (markusthoemmes) [inline tools/ubuntu-setup/pip.sh]: The dependency only strikes on usage of ansible, so the order doesn't matter here i suppose.

**Pull request closed without merging**